website/integrations/services/aws/index.md

```diff
@@ -75,3 +82,94 @@ To use the user's username, use this snippet
+
+-   In AWS Navigate to: `IAM Identity Center -> Settings -> Identity Source (tab)`
+-   On the right hand side click `Actions -> Change identity source`
+-   Select `External Identity Provider`
+-   Under `Service Provider metadata` it'll allow you to download a metadata file. Download this file.
+
+Now go to your authentik instance, and perform the following steps.
+
+-   Under _Providers_ create a new _SAML Provider from metadata_. Give it a name, and upload the metadata file AWS gave you.
+-   Click _Next_. Give it a name, and close.
+-   If you haven't done so yet, create an application for AWS and connect the Provider to it.
+-   Navigate to the provider you've just created, select _Edit_
+-   Copy the _Issuer URL_ to the _Audience_ field.
+-   Under _Advanced Protocol Settings_ set a _Signing Certificate_
+-   Save and Close.
+-   Under _Related Objects_ download the _Metadata file_, and the _Signing Certificate_
+
+Now go back to your AWS instance
+
+-   Under `Identity provider metadata` upload both the the `Metadata file` and `Signing Certificate` authentik gave you.
+-   Choose `Next`.
+-   In your settings pane, under the tab `Identity Source` click `Actions -> Manage Authentication`
+-   Take note of the `AWS access portal sign-in URL` (this is especially important if you've changed it from the default)
+
+Now go back to your authentik instance.
+
+-   Navigate to the Application you've created for AWS and click _Edit_.
+-   Under _UI Settings_ make sure the _Start URL_ matches the _AWS access portal sign-in URL_
+
+## Caveats and Troubleshooting
+
+-   Users need to exist in AWS in order to use them through authentik. AWS will throw an error if it doesn't recognise the user.
+-   In case you're stuck, you can see the SSO logs in Amazon CloudTrail -> Event History. Look for `ExtenalIdPDirectoryLogin`
+
+Note:
+
+## Optional: Auto provisioning with SCIM
+
+Some people may opt for the automatic provisioning feature called SCIM.
+SCIM allows you to synchronise (part of) your directory to AWS's IAM. Saving you the hassle of having to create users by hand.
+In order to do so, go to your AWS Identity Center
+
+-   In your `Settings` pane, locate the `Automatic Provisioning` Info box. Click `Enable`
+-   AWS will give you an `SCIM Endpoint` and a `Access Token`. Take note of these
+
+Go back to your authentik instance
+
+-   Navigate to _Providers_ -> _Create_
+-   Select _SCIM Provider_
+-   Give it a name, under _URL_ enter the _SCIM Endpoint_ and under _Token_ enter the _Access Token_ AWS provided you with.
+-   In case you wish, change the user filtering settings to your liking. Click _Finish_
+
+-   Go to _Customization -> Property Mappings_
+-   Click _Create -> SCIM Mapping_
+-   Make sure to give the mapping a name that's lexically lower than `authentik default`, for example `AWS SCIM User mapping`
+-   As the expression, enter:
+
+```python
+# The only thing this does, is strip the default mapping from it's 'photos' attribute,
```

Comment:
```suggestion
# This expression strips the default mapping from its 'photos' attribute,
```

Reply:
@justSem please double-review this suggestion to make sure I did not change the technical accuracy when I changed the grammar. ;-)

Reply:
This is correct!